.github/workflows/tests.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         cd rules/
         for changed_file in ${{ steps.files.outputs.added_modified }} ${{ steps.files.outputs.renamed }}; do
           if [[ ! $changed_file =~ .git|.md ]]; then
-            tag=$(grep 'name:' $changed_file | sed 's/^.*: //')
+            tag=$(grep '\sname:' $changed_file | sed 's/^.*: //')
```

Comment:
im not sure if this needs to be escaped further. im inclined to test in prod, but what does the reviewer think?

Reply:
```suggestion
            tag=$(grep '\\sname:' $changed_file | sed 's/^.*: //')
```

not sure

Reply:
i dont think further quoting is needed here

https://yaml-multiline.info/